src/restaurant_inspections_api/handlers/businesses.clj

```diff
@@ -13,6 +13,13 @@
                              :perPage (validate/per-page per-page)
                              :page (validate/page page)}))
 
+(defn format-params
+  [params-map]
+  (assoc params-map
+         :zipCodes (when-let [zipCodes (:zipCodes params-map)]
```

Comment:
I think is not: https://github.com/Code-for-Miami/restaurant-inspections-api/pull/64/files#diff-c94f3fb3ea7f62383b2dcd1fe66f2d94R90